conversations/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "conversations",
+  "dependencies": {}
```

Comment:
I'm not sure if this requires a specific `twilio` version that's not the default on Functions. If so, can you please mark that here?